util/types/mydecimal.go

```diff
@@ -159,6 +159,9 @@ func digitsToWords(digits int) int {
 	return (digits + digitsPerWord - 1) / digitsPerWord
 }
 
+// MyDecimalStructSize is the struct size of MyDecimal.
+const MyDecimalStructSize = 40
```

Comment:
`unsafe.Sizeof(MyDecimal{})` ?

Reply:
It will be more clear how large `MyDecimal` is, and we should not change the struct of the `MyDecimal`.